mlir/lib/Dialect/Rock/utility/AccelEmitter.cpp

```diff
@@ -130,27 +131,110 @@ Value AccelEmitter::generateThreadwiseViewBufferC(PatternRewriter &b,
   return viewC;
 }
 
+// **************************
```

Comment:
Weird comment format

Reply:
... though that's file style. Maybe worth fixing to the `//=---------------=//` style seen elsewhere in the codebase both here and below